sagemaker-python-sdk/scikit_learn_model_registry_batch_transform/scikit_learn_model_registry_batch_transform.ipynb

```diff
@@ -0,0 +1,613 @@
+{
```

Comment:
<p>nit: You could potentially refactor this into a small utility function into which you could pass an estimator as param and re-use it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3086/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
You are right, but in this case, it is important to show the SDK. I added more details about how to view Model Groups and versions, which I think is important here, as customers tend to think that Model Registry is a feature of SageMaker Studio.